fix for compiler error ""error: more than one instance of overloaded function isnan matches the argument list"
author: Kenneth Hoste (HPC-UGent)
--- gate_v7.1/source/digits_hits/src/GateToRoot.cc.orig	2016-03-03 19:41:50.569854028 +0100
+++ gate_v7.1/source/digits_hits/src/GateToRoot.cc	2016-03-03 19:43:09.370590639 +0100
@@ -765,7 +765,7 @@
       if (dzg1 > dzg2) {dev = acos(-dev)*180/Pi;}
       else {dev = acos(dev)*180/Pi - 180;}
 
-      if (isnan(dev) ) dev = 0.;
+      if (std::isnan(dev) ) dev = 0.;
 
       // G4cout<< " dev = " << dev << G4endl;
 
